src/utilities/resizeObserverHelper.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * ResizeObserverHelper
+ * @description Helper class for ResizeObserver
+ * @param callback - Callback function on resize
+ */
+
+export class ResizeObserverHelper {
```

Comment:
question: why are we using a class instead of just a composable?

Reply:
So it looks close to native ResizeObserver API implementation. With native one you would do:
```
resizeObserver.value = new ResizeObserver()
resizeObserver.value.observe(...)
...
resizeObserver.value.unobserve(...)
```

With this helper wrapper it's only slightly different:
```
resizeObserver.value = ResizeObserverHelper.create()
resizeObserver.value.observe(...)
...
resizeObserver.value.unobserve(...)
```

Reply:
Done